test/lib/ufe/testGroupCmd.py

```diff
@@ -35,6 +34,38 @@
 import os
 import unittest
 
+def createStage():
+    ''' create a simple stage '''
+    cmds.file(new=True, force=True)
+    import mayaUsd_createStageWithNewLayer
+    proxyShapePathStr = mayaUsd_createStageWithNewLayer.createStageWithNewLayer()
+    proxyShapes = cmds.ls(type="mayaUsdProxyShapeBase", long=True)
+    proxyShapePath = proxyShapes[0]
+    proxyShapeItem = ufe.Hierarchy.createItem(ufe.PathString.path(proxyShapePath))
+    proxyShapeContextOps = ufe.ContextOps.contextOps(proxyShapeItem)
+    stage = mayaUsd.lib.GetPrim(proxyShapePath).GetStage()
+    return (stage, proxyShapePathStr, proxyShapeItem, proxyShapeContextOps)
+
+class SphereGenerator():
+    ''' simple sphere generator '''
+    def __init__(self, num, contextOp, proxyShapePathStr):
+        self.gen = self.__generate(num)
+        self.num = num
```

Comment:
Wonder why we need this limit.

Reply:
Not enough value in spending time on this.

Reply:
@ppt-adsk  thanks!